File: Famoser.SyncApi/Api/Communication/Response/AuthorizationResponse.cs
using Famoser.SyncApi.Api.Communication.Entities;
using Famoser.SyncApi.Api.Communication.Response.Base;

namespace Famoser.SyncApi.Api.Communication.Response
{
    public class AuthorizationResponse : BaseResponse
    {
        public UserEntity UserEntity { get; set; }
        public DeviceEntity DeviceEntity { get; set; }
    }
}

File: Famoser.SyncApi.Webpage/src/Famoser/SyncApi/Models/Communication/Response/AuthorizationResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 04.11.2016
 * Time: 19:20
 */

namespace Famoser\SyncApi\Models\Communication\Response;


use Famoser\SyncApi\Models\Communication\Entities\DeviceCommunicationEntity;
use Famoser\SyncApi\Models\Communication\Entities\UserCommunicationEntity;
use Famoser\SyncApi\Models\Communication\Response\Base\BaseResponse;

/**
 * the response to an AuthorizationRequest
 * @package Famoser\SyncApi\Models\Communication\Response
 */
class AuthorizationResponse extends BaseResponse
{
    /* @var UserCommunicationEntity $UserEntity */
    public $UserEntity;

    /* @var DeviceCommunicationEntity $DeviceEntity */
    public $DeviceEntity;
}
